src/infrastructure/todoapp-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: todoapp:3.0.0
+ namespace: todoapp
+ labels:
+   app: todoapp
+spec:
+ containers:
+ - name: todoapp
+   image: dumavb/todoapp3
```

Comment:
The image name does not match the specified format `{yourname}/todoapp:3.0.0`. Please update it to follow this format.